wine/PKGBUILD

```diff
@@ -0,0 +1,157 @@
+  gettext         #lib32-gettext
+  libpcap         #lib32-libpcap
+  libunwind       #lib32-libunwind
+  libxcursor      #lib32-libxcursor
+  libxkbcommon    #lib32-libxkbcommon
+  libxi           #lib32-libxi
+  libxrandr       #lib32-libxrandr
+  wayland         #lib32-wayland
+)
+makedepends=(autoconf bison perl flex mingw-w64-gcc
+  git
+  alsa-lib              #lib32-alsa-lib
+  ffmpeg
+  gnutls                #lib32-gnutls
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  libcups               #lib32-libcups
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  libxxf86vm            #lib32-libxxf86vm
+  mesa                  #lib32-mesa
+  mesa-libgl            #lib32-mesa-libgl
+  opencl-headers
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  pcsclite              #lib32-pcsclite
+  python
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  unixodbc
+  v4l-utils             #lib32-v4l-utils
+  vulkan-headers
+  vulkan-icd-loader     #lib32-vulkan-icd-loader
+)
+makedepends+=(
+  ntsync-header
+)
+optdepends=(
+  alsa-lib              #lib32-alsa-lib
+  alsa-plugins          #lib32-alsa-plugins
+  cups                  #lib32-libcups
+  dosbox
+  ffmpeg
+  gnutls                #lib32-gnutls
+  gst-plugins-bad
+  gst-plugins-base      #lib32-gst-plugins-base
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  gst-plugins-good      #lib32-gst-plugins-good
+  gst-plugins-ugly
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  pcsclite              #lib32-pcsclite
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  unixodbc
+  v4l-utils             #lib32-v4l-utils
+  wine-gecko
+  wine-mono
+)
+optdepends+=(
+  NTSYNC-MODULE
+)
+makedepends=(${makedepends[@]} ${depends[@]})
+install=wine.install
+
+prepare() {
+  # Get rid of old build dirs
+  rm -rf $pkgname-{32,64}-build
+  mkdir $pkgname-{32,64}-build
+
+  cd wine
+
+  patch -Np1 -i "$srcdir"/wine-ntsync.patch
+
+  ./tools/make_requests
+  ./dlls/winevulkan/make_vulkan
+  ./tools/make_specfiles
+  autoreconf -fiv
+}
+
+build() {
+  # Doesn't compile without remove these flags as of 4.10
+  export CFLAGS="$CFLAGS -ffat-lto-objects"
+
+  # Apply flags for cross-compilation
+  export CROSSCFLAGS="-O2 -pipe -g"
+  export CROSSCXXFLAGS="-O2 -pipe -g"
+  export CROSSLDFLAGS="-Wl,-O1"
+
+  echo "Building Wine-64..."
+  cd "$srcdir/$pkgname-64-build"
+  ../wine/configure \
+    --prefix=/usr \
+    --libdir=/usr/lib \
+    --with-x \
+    --with-wayland \
+    --with-gstreamer \
+    --with-mingw \
+    --enable-archs=x86_64,i386
+
+  make
+}
+
+package() {
+  echo "Packaging Wine-64..."
+  cd "$srcdir/$pkgname-64-build"
+  make prefix="$pkgdir/usr" \
+    libdir="$pkgdir/usr/lib" \
+    dlldir="$pkgdir/usr/lib/wine" install
+
```

Comment:
There's no strip. Maybe it was intended, but I prefer to clarify.

Reply:
It is following the Arch PKGBUILD in what to do with debugging symbols. For reference: https://gitlab.archlinux.org/archlinux/packaging/packages/wine/-/merge_requests/2/diffs

Reply:
Can we not just disable it? I dont think we want to have a 600MB bigger package, till we have PE strip from pacman

Reply:
> Can we not just disable it? I dont think we want to have a 600MB bigger package, till we have PE strip from pacman

Well, right way to go here is to use ``pestrip`` [1], but it's still not pushed to pacman package, so we're gonna be a bit stuck with that for a while.

- https://gitlab.archlinux.org/pacman/libmakepkg-dropins/-/blob/main/tidy/pestrip.sh

Reply:
> Can we not just disable it? I dont think we want to have a 600MB bigger package, till we have PE strip from pacman

The point of the MR I linked is to have a package that can allow users to report breakage to upstream Wine. Yeah, might be adding ntsync on top, but we add what is MR'd to wine, in an unmodified state too. So I think our Wine/Wine-staging should at this point in time facilitate a good base for issues to be reported to the upstream projects.